packages/manager/src/features/linodes/LinodesDetail/LinodeNetworking/LinodeNetworking.tsx

```diff
@@ -419,11 +493,18 @@ class LinodeNetworking extends React.Component<CombinedProps, State> {
   }
 
   renderErrorState = () => {
-    const { IPRequestError } = this.state;
-    const errorText = IPRequestError
-      ? IPRequestError
-      : 'There was an error retrieving your networking information.';
-    return <ErrorState errorText={errorText} />;
+    const { IPRequestError, ipv6Error } = this.state;
+    let errorText;
+    if (IPRequestError) {
+      errorText =
+        IPRequestError ||
+        'There was an error retrieving your networking information.';
+    } else if (ipv6Error) {
+      errorText =
+        ipv6Error ||
+        'There was an error retrieving your networking information.';
+    }
+    return errorText ? <ErrorState errorText={errorText} /> : null;
```

Comment:
This function is only called when there is an error so we can simplify this to
```suggestion
    const errorText =
      IPRequestError ||
      ipv6Error ||
      'There was an error retrieving your networking information.';
    return <ErrorState errorText={errorText} />;
```